app/ui-react/packages/ui/src/Integration/Metrics/IntegrationDetailMetrics.tsx

```diff
@@ -35,7 +35,7 @@ export class IntegrationDetailMetrics extends React.Component<
   > {
   public render() {
     const okMessagesCount = this.props.messages! - this.props.errors!;
-    const startAsDate = new Date(this.props.start!);
+    const startAsDate = new Date(this.props.start! * 1000);
```

Comment:
Perhaps we can do this on the server (checkout the comments on PR)

Reply:
Done, thanks.

Reply:
Done.